h2o-core/src/main/java/water/H2O.java

```diff
@@ -298,6 +301,9 @@ public static void printHelp() {
     /** -login_conf is login configuration service file on local filesystem */
     public String login_conf = null;
 
+    /** -ssl_config path (absolute or relative) to a file containing all SSL related configurations */
+    public String ssl_config = null;
```

Comment:
❗ After discussion with @tomkraljevic - he proposed to have a single option which will enable/disable SSL - the SSL configuration will be auto-generated. Does it make sense? 
Is it possible? Do we have use-case showing the case when people would like to pass their own configuration?